DEVELOPERS.md

```diff
@@ -1,4 +1,6 @@
-Information for Code Climate CLI developers and contributors.
+# Development
```

Comment:
WDYT about disabling this check. All three cases where you listened to it feel worse to me.